src/components/EditForm.tsx

```diff
@@ -13,75 +13,36 @@ interface EditFormProps {
   updateTask: (task: Task) => void;
 }
 
-const EditForm:Component = (props: EditFormProps) => {
+const EditForm: Component<EditFormProps> = (props) => {
   const [updatedTaskName, setUpdatedTaskName] = createSignal("");
-  let dialogRef;
+  let dialogRef: HTMLDialogElement;
 
-  const handleFormSubmit = (e) => {
+  const handleFormSubmit = (e: SubmitEvent) => {
```

Comment:
I would reccommend that if you don't use a function multiple times, you just put it where it will end up being. So if you just put this function anonymously into the `onSubmit` event handler, the type for `e` will be implicitly derived and you won't have to worry about figuring out and declaring the type for it.